src/main.js

```diff
@@ -12,30 +10,21 @@ var viewSavedCoversButton = document.querySelector('.view-saved-button');
 var createBookBtn = document.querySelector('.create-new-book-button');
 var homeView = document.querySelector('.home-view');
 var savedView = document.querySelector('.saved-view');
+var savedCoversSection = document.querySelector('.saved-covers-section');
 var makeNewButton = document.querySelector('.make-new-button');
 var formView = document.querySelector('.form-view');
 var userCover = document.querySelector('.user-cover');
 var userTitle = document.querySelector('.user-title');
 var userDecr1 = document.querySelector('.user-desc1');
 var userDecr2 = document.querySelector('.user-desc2');
 
-var savedCoversSection = document.querySelector('.saved-covers-section');
-
-// We've provided a few variables below
 var savedCovers = [
   createCover("http://3.bp.blogspot.com/-iE4p9grvfpQ/VSfZT0vH2UI/AAAAAAAANq8/wwQZssi-V5g/s1600/Do%2BNot%2BForsake%2BMe%2B-%2BImage.jpg", "Sunsets and Sorrows", "sunsets", "sorrows")
 ];
 var currentCover;
-var currentView;
```

Comment:
I deleted the only place it was being called on 35 too - if you look below, it's gone. 
The reason I included it in the first place is because there's probably an opportunity to dry up our code by including as a parameter the view we're coming from on any button click so we would just need to worry about hiding that view.

Reply:
Ohhhh I see, awesome!